mutiny-core/src/nodemanager.rs

```diff
@@ -926,13 +926,19 @@ impl<S: MutinyStorage> NodeManager<S> {
         let onchain = self.list_onchain()?;
         let mut activity = Vec::with_capacity(lightning.len() + onchain.len());
         for ln in lightning {
-            activity.push(ActivityItem::Lightning(Box::new(ln)));
+            // Only show paid invoices
+            if ln.paid {
+                activity.push(ActivityItem::Lightning(Box::new(ln)));
+            }
         }
         for on in onchain {
             activity.push(ActivityItem::OnChain(on));
         }
 
         activity.sort();
+        // Newest first
```

Comment:
```
activity.sort_by(|a, b| b.cmp(a));
```

Reply:
We could also change it so the `impl ord` sorts in descending order.

Reply:
I'm fine with tony's solution